qiskit_experiments/library/calibration/fine_amplitude.py

```diff
@@ -146,7 +146,14 @@ def update_calibrations(self, experiment_data: ExperimentData):
 class FineXAmplitudeCal(FineAmplitudeCal):
     """A calibration experiment to calibrate the amplitude of the X schedule."""
 
-    __analysis_class__ = FineXAmplitudeAnalysis
+    @functools.wraps(FineAmplitudeCal.__init__)
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.analysis.set_options(
```

Comment:
Is this temporarily? this looks bit awkward. Probably we need to add dedicated analysis to each experiment?

Reply:
Yeah i should update this to actually copy the init args